doc/index.rst

```diff
@@ -28,6 +28,12 @@ Important classes:
 * `BendModeToForce` transforms the bending mode to actuator forces and vice versa.
 * `SubSysAdap` transforms the degree of freedom or actuator forces from ZEMAX coordinate to subsystem's coordinate and vice versa.
 
+.. uml:: uml/ctrlIntfClass.uml
+    :caption: OFCCalculation class diagram.
```

Comment:
This class diagram is more like the package of `ctrlIntf` instead of `OFCCalculation` itself.